cpp/cmake/thirdparty/get_cucollections.cmake

```diff
@@ -14,14 +14,16 @@
 # limitations under the License.
 #=============================================================================
 
+# cuCollections doesn't have a version
+
 function(find_and_configure_cucollections)
 
     # Find or install cuCollections
     rapids_cpm_find(cuco 0.0
         GLOBAL_TARGETS cuco::cuco
         CPM_ARGS
-            GITHUB_REPOSITORY NVIDIA/cuCollections
-            GIT_TAG           0d602ae21ea4f38d23ed816aa948453d97b2ee4e
+            GITHUB_REPOSITORY PointKernel/cuCollections
+            GIT_TAG           static-multi-map
```

Comment:
What is the ETA on these changes being upstreamed to cuCollections?

Reply:
@jrhemstad may know the answer.

Reply:
Soon(TM)

Reply:
@PointKernel update this to the git hash for the multimap merge.